README.md

```diff
@@ -453,6 +459,16 @@ You can checkout these links on this subject:
 - https://webpack.js.org/configuration/dev-server/#devserver-hot
 - https://webpack.js.org/guides/hmr-react/
 
+If you are using hot reloading, you should either set the `dev_server/hot_reloading` key to TRUE or set the 
+ENV value `WEBPACKER_HMR=TRUE`. That way, your stylesheet pack tag will do **nothing** because you
+need your styles inlined in your JavaScript for hot reloading to work properly.
```

Comment:
Hi @dhh. The issue I'm trying to solve is:

* We can't put Ruby in the YML file, since the YML file is read by JavaScript.
* On bigger teams, it's common to have some developers wishing to do hot reloading or the webpack-dev-server, while others prefer just to have the files built in a similar manner to production.
* We don't want some developers on a team to have the webpack.yml file modified just for their own environment.
* We want a team to have different Procfiles depending if a developer wants to use these options for hot-reloading and the webpack-dev-server.